qiskit_experiments/library/calibration/fine_drag.py

```diff
@@ -0,0 +1,251 @@
+                measured signal to the interval [0, 1]. Defaults to True.
+        """
+        options = super()._default_experiment_options()
+        options.repetitions = list(range(20))
+        options.schedule = None
+
+        return options
+
+    @classmethod
+    def _default_analysis_options(cls) -> Options:
+        """Default analysis options."""
+        options = super()._default_analysis_options()
+        options.normalization = True
+        options.angle_per_gate = 0.0
+        options.phase_offset = -np.pi / 2
+        options.amp = 1.0
+
+        return options
+
+    def __init__(self, qubit: int):
+        """Setup a fine amplitude experiment on the given qubit.
+
+        Args:
+            qubit: The qubit on which to run the fine amplitude calibration experiment.
+        """
+        super().__init__([qubit])
+
+    @staticmethod
+    def _pre_circuit() -> QuantumCircuit:
+        """Return the quantum circuit to apply before repeating the Rp - Rz - Rp - Rz gates."""
+        return QuantumCircuit(1)
+
+    @staticmethod
+    def _post_circuit() -> QuantumCircuit:
+        """Return the quantum circuit to apply after repeating the Rp - Rz - Rp - Rz gates."""
+        circ = QuantumCircuit(1)
+        circ.ry(np.pi / 2, 0)  # Maps unwanted Z rotations to qubit population.
+        return circ
+
+    def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
+        """Create the circuits for the fine DRAG calibration experiment.
+
+        Args:
+            backend: A backend object.
+
+        Returns:
+            A list of circuits with a variable number of gates. Each gate has the same
+            pulse schedule.
+        """
+        schedule, circuits = self.experiment_options.schedule, []
+
+        drag_gate = Gate(name=schedule.name, num_qubits=1, params=[])
+
+        for repetition in self.experiment_options.repetitions:
+            circuit = self._pre_circuit()
+
+            for _ in range(repetition):
+                circuit.append(drag_gate, (0,))
+                circuit.rz(np.pi, 0)
+                circuit.append(drag_gate, (0,))
+                circuit.rz(np.pi, 0)
+
+            circuit.compose(self._post_circuit(), inplace=True)
+
+            circuit.measure_all()
+            circuit.add_calibration(schedule.name, self._physical_qubits, schedule, params=[])
+
+            circuit.metadata = {
+                "experiment_type": self._type,
+                "qubits": self.physical_qubits,
+                "xval": repetition,
+                "unit": "gate number",
+            }
+
+            circuits.append(circuit)
+
+        return circuits
+
+
+class FineXDrag(FineDrag):
+    """Class to fine calibrate the DRAG parameter of an X gate.
+
+    # section: see_also
+        qiskit_experiments.library.calibration.fine_drag.FineDrag
+    """
+
+    @staticmethod
+    def _pre_circuit() -> QuantumCircuit:
```

Comment:
Why override this method without changing the? I still think it is good to have this class so we can add the right schedule retrieval and updater logic once the cal experiment classes can do those things.